itchiodl/downloader/__main__.py

```diff
@@ -15,16 +19,19 @@ def main():
     parser.add_argument(
         "-p",
         "--platform",
-        help="Platform to download for (default: all), will accept values like 'windows', 'linux', 'osx' and android",
+        help=(
+            "Platform to download for (default: all), will accept values like 'windows', 'linux', "
+            "'osx' and 'android'"
+        ),
     )
 
     parser.add_argument(
         "--human-folders",
-        type=bool,
-        default=False,
-        const=True,
-        nargs="?",
-        help="Download Folders are named based on the full text version of the title instead of the trimmed URL title",
+        action='store_true',
```

Comment:
should be double quotes

Reply:
Sorry I should have noticed the failed workflow. I have fixed that.

I've also noticed the CLI tools were not being checked in the workflow, so I updated that.